tests/scenario_8/README.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+```
++-----------+
+|           |-----------[ VM 1 ] (vni 42)
+| Butterfly |
+|           |-----------[ VM 2 ] (vni 42)
++-----------+
+
+```
+
+This scenario just connects two virtual machines on the same butterfly
+on the different network (vni) and make a bunch of network tests (sent udp
```

Comment:
"a bunch of tests", doesn't explain which tests this scenario do.
Actually, I'm not sure that a readme per scenario is really useful, I mean this readme as it is now, does not explain how the scenario work, and the scripts as they are today are small enough to explain what they are doing by them-self.